docker/docker-compose.yml.py

```diff
@@ -79,6 +79,28 @@ def mail_config(num: int, name: str) -> Dict[str,Any]:
 
     return data
 
+def redis_config(num: int, name: str) -> Dict[str,Any]:
+    data: Dict[str, Any] = dict()
+    data['image'       ] = 'redis:6'
+    data['hostname'    ] =  name
+    data['ports'       ] = [ 6379 ]
+    data['networks'    ] = {
+        'scznet': {
+            'ipv4_address': f'{subnet}.{num}',
+            'aliases':      [ f'{name}.vm.{domain}' ]
+        }
+    }
+    data['extra_hosts'] = [ f'{h}.{domain}:{subnet}.{hosts["lb"]}' for h in logical_hosts ]
+    data['healthcheck'] = {
+        'test': [ 'CMD', '/usr/bin/test', '!', '-e', '/etc/nologin' ],
+        'interval': '5s',
+        'timeout': '1s',
+        'retries': '1',
+        'start_period': '0s'
+    }
+
+    return data
+
```

Comment:
Kun je voor de redis changes een aparte PR maken?  Die wil is sowieso op korte termijn mergen.